mapper.go

```diff
@@ -24,10 +24,11 @@ import (
 )
 
 var (
-	identifierRE = `[a-zA-Z_][a-zA-Z0-9_]+`
-	metricLineRE = regexp.MustCompile(`^(\*\.|` + identifierRE + `\.)+(\*|` + identifierRE + `)$`)
-	labelLineRE  = regexp.MustCompile(`^(` + identifierRE + `)\s*=\s*"(.*)"$`)
-	metricNameRE = regexp.MustCompile(`^` + identifierRE + `$`)
+	identifierRE      = `[a-zA-Z_][a-zA-Z0-9_]*`
```

Comment:
Now that this is only used in one place, just inline it into the regex where it's used, I'd say.